scripts/package.json

```diff
@@ -10,8 +10,8 @@
     "docu:container:generate:container-api": "documentation readme ../container/typings/LuigiContainer.svelte.d.ts --parse-extension ts -f md --readme-file=../docs/luigi-container-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
     "docu:container:generate:compound-container-api": "documentation readme ../container/typings/LuigiCompoundContainer.svelte.d.ts --parse-extension ts -f md --readme-file=../docs/luigi-compound-container-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
     "docu:client": "npm run docu:client:validate && npm run docu:client:generate:section",
-    "docu:client:generate:section": "documentation readme ../client/src/luigi-client.js -f md --readme-file=../docs/luigi-client-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
-    "docu:client:validate": "documentation lint ../client/src/luigi-client.js",
+    "docu:client:generate:section": "documentation readme ../client/luigi-client.d.ts --parse-extension ts -f md --readme-file=../docs/luigi-client-api.md --section=\"API Reference\" --markdown-toc=false --quiet --github false",
+    "docu:client:validate": "documentation lint ../client/luigi-client.d.ts",
```

Comment:
It seems to be some issue with parsing TS files that comes from documentation lib - according to their changelog it was fixed in version 14.0.2:
https://github.com/documentationjs/documentation/blob/master/CHANGELOG.md#bug-fixes-1

I tested the latest version locally and it can handle TS imports correctly - but unfortunatelly many newlines are removed and output file is unreadable :( Similar issue is already reported:
https://github.com/documentationjs/documentation/issues/1595

I can see some possible solutions here, but none of them is perfect:
1. fork documentation lib from lower version and try to fix TS issue on our own
2. freeze building of documentation file for client until TS issues are fixed - API is not updated so often

Reply:
I found a workaround to handle docu issue - for now each TS file needs to be documented separately